src/main/java/Duke.java

```diff
@@ -1,10 +1,79 @@
+import java.util.Scanner;
+
 public class Duke {
+    public static String boundary = "____________________________________________________________" + System.lineSeparator();
+    public static Task[] taskList = new Task[100];
+    public static int countTask = 0;
+
+    public static void printList() {
+        System.out.println(boundary + "Here are the tasks in your list:");
+        for (int i = 0; i < countTask; i ++) {
```

Comment:
In the for loop, you could remove the space in "i ++", such that it follows this suggested coding guideline in the se-edu website:
![image](https://user-images.githubusercontent.com/58351701/152276155-07358ba7-f8c2-4d33-bb4a-bc7719e96c48.png)